### ISSUE TITLE
Add way to view preview of other documents

### ISSUE THREAD

**Issue by alexmigf** [wpovernight/woocommerce-pdf-invoices-packing-slips#268]
Currently the Preview only works with Invoice, would be good to see also for the other documents.

**Comment by Spreeuw** [comment]
I think the MVP version of this would be to simply have the document selector in the Documents settings to change the document type, @alexmigf that would be quite simple to implement already I think?

**Comment by alexmigf** [comment]
What you think of switching document in settings and that also changes to the correspondent preview document? Because we already have a selector there for documents:

![Captura de ecrã de 2022-01-27 08-42-28](https://user-images.githubusercontent.com/533273/151322943-a27d7ec7-76f1-436c-9b05-bf141a800300.png)

**Comment by Spreeuw** [comment]
@alexmigf see my comment above yours, looks like our messages crossed :)

**Comment by Terminator-Barbapapa** [comment]
I already added a document selector in the preview panel. At the moment it does not trigger the correct document yet though. But for the MVP I think using the document selector in the Documents settings would work fine. Should I remove my my document selector?

**Comment by Spreeuw** [comment]
Does that mean there are 2 dropdowns in the Document settings now? In that case I think we should hide the "preview" document selector on that page. I don't see why it should be removed if you have already done the work!

**Comment by Terminator-Barbapapa** [comment]
It's located next to the order search.

<img width="478" alt="Schermafbeelding 2022-01-27 om 10 25 36" src="https://user-images.githubusercontent.com/19350150/151330395-e24211b7-e58e-486b-b6b4-4d6a4a6bfe6b.png">

**Comment by Spreeuw** [comment]
Yes, so we should probably hide that on the Documents page?

**Comment by alexmigf** [comment]
> Yes, so we should probably hide that on the Documents page?

And use it when not in Documents settings page?

**Comment by Spreeuw** [comment]
yes, otherwise there's no point in having it in the first place 😆

**Comment by Spreeuw** [comment]
This doesn't work as expected. It reloads on every click (making it seem like the document is being changed), but it only actually changes when I click on the text itself.

**Comment by Spreeuw** [comment]
✅ Seems to work smoothly now. We need a bit more intelligence for the Credit Note preview, but I suggest to open a separate issue for that @alexmigf ?

**Comment by alexmigf** [comment]
Yes, I will do it.
